src/substrate/filters/enricher.ts

```diff
@@ -36,22 +36,25 @@ export async function Enrich(
        */
       case EventKind.NewSession: {
         const [ sessionIndex ] = event.data as unknown as [ SessionIndex ] & Codec
-        const tmp_validators = await api.query.session.validators<Vec<ValidatorId>>();
+        const validators = await api.derive.staking.validators();
```

Comment:
Has this been tested on both edgeware and kusama? Need to ensure the derive calls don't fail.

Reply:
Yes I have tested it, derive calls work on both kusama and edgeware.
The left terminal in the image is output for ```URL_KUSAMA``` and right is for ```URL_EDG```

![image](https://user-images.githubusercontent.com/35498810/91188713-0d2e0e00-e70b-11ea-8daf-8d7c8b376c47.png)